### ISSUE TITLE
Inquiry about CUDA Acceleration for Dedalus

### ISSUE THREAD

**Issue by whpy** [DedalusProject/dedalus#282]
**Description:**

**Background:**
I've been using Dedalus for active matter simulation. I've found it to be a valuable tool, but I'm also interested in exploring ways to improve its performance, particularly for computationally intensive tasks.

**Request:**
I'm interested in knowing if there are any plans or considerations for developing a CUDA-accelerated version of Dedalus. I want to utilize the GPU resources in my PC and our lab.

**Comment by kburns** [comment]
Yes this is a major component of my planned work over the next year, but I expect it will take a while before a prototype is available.

**Comment by whpy** [comment]
> Yes this is a major component of my planned work over the next year, but I expect it will take a while before a prototype is available.

Thanks for your reply!
